packages/ssz/src/type/container.ts

```diff
@@ -551,7 +551,7 @@ export function precomputeJsonKey<Fields extends Record<string, Type<unknown>>>(
   if (casingMap) {
     const keyFromCaseMap = casingMap[fieldName];
     if (keyFromCaseMap === undefined) {
-      throw Error(`casingMap[${fieldName}] not defined`);
+      throw Error(`casingMap[${String(fieldName)}] not defined`);
```

Comment:
why is this required? fieldName should be type `string`

Reply:
Hi @nflaig when I ran the build command this error was showing up "Implicit conversion of a 'symbol' to a 'string' will fail at runtime. Consider wrapping this expression in 'String(...)'. " to overcome this I applied typecasting